### PR TITLE
Ansible collection proposal

### PR DESCRIPTION
**Pull request by mbwhite**
Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

### CODE REVIEW

File: labs/fabric-ansible-collection.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+# Initial Committers
+- https://github.com/mbwhite
+
+# Sponsor
+
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
+- https://github.com/<your_repo>
```
Comment (tkuhrt) [inline labs/fabric-ansible-collection.md:43]:
```suggestion
```

**Review by tkuhrt — COMMENTED** [review]
Hi, @mbwhite 

This sounds like an interesting lab. I also wonder if there is some collaboration that can be done with the Hyperledger Bevel community. I am not sure if Bevel is moving away from Ansible or not in their latest roadmap, but might be worth a conversation to see if this work fits.

**Comment by mbwhite** [comment]
Hello @tkuhrt .. one thing that's been on the to-do list has been to look at Bevel. At the very least as Fabric maintainers, we should have an 'open line of communication' so when things change with say with Docker images we can all be pro-active.

Do you know a the best  contact to open the conversation?  

There are variety of different technologies that do the 'Ansible' job, we've seen it used extensively and whilst it has its faults (doesn't everything!)  it does the job.

**Comment by tkuhrt** [comment]
I should have tagged @sownak in my first message. Sownak, I wanted to connect you with @mbwhite. This PR is about the ansible collection that exists for Fabric deployments. Hoping to get collaboration between what is being done in Hyperledger Bevel and this proposal.

**Comment by tkuhrt** [comment]
Also, if it is easier @mbwhite to have a conversation with the Bevel folks, you could check out the #bevel-contributors channel on the [Hyperledger Chat](https://discord.gg/hyperledger)

**Comment by denyeart** [comment]
I agree the discussion with Bevel is a good idea.
I would also agree to this lab as proposed and could be a sponsor. This ansible collections works with fabric-operator which is already a lab, so makes sense to manage the ansible collection as a lab together with fabric-operator.

**Comment by sownak** [comment]
Is the ansible-collection working with fabric-operator or is it independently deploying Fabric?

**Comment by mbwhite** [comment]
Hi @sownak... *today* it uses the FabricOperationsConsonle & FabricOperator to deploy and create Peers etc; once they are created it's generic administration.

My intention though would be to aim to make it generic so it's possible to use any operator.  The vast majority of the codebase is for checks relating to whether the requested configuration is valid, does the resource need creating or just updating.

**Review by tkuhrt — APPROVED** [review]
I am approving this lab. Long term, I would like to see the conversations continue with Bevel.

**Review by knagware9 — APPROVED** [review]
Its an interesting lab, discussion & collaboration with Bevel and Operator lab should be taken forward.

**Comment by mbwhite** [comment]
@tkuhrt and @knagware9 thank you I would very much like there to be a more unified story to the production deployment. 

What's the best route of taking the existing repo from where it is? Does it need to be cleaned up (i.e. remove the IBM bits) and then moved or can it be cleaned in place?

**Comment by tkuhrt** [comment]
@mbwhite : That is completely up to you. We have options for either transferring the repo or creating a new repo at https://github.com/hyperledger-labs/governance/issues/new/choose. I can merge this PR given the approvals from two lab stewards. If you link this PR in the governance issue, we will be able to see whether you choose to create new or transfer.

**Comment by mbwhite** [comment]
thanks @tkuhrt  my preference would be a transfer and update-in-labs.  I'll raise a governance issue.

**Pull request merged**